documentation/functions/domain/LOC.md

```diff
@@ -11,6 +11,24 @@ parameters:
   - size
   - horizontal_precision
   - vertical_precision
+parameter_types:
+  name: string
+  target: string
+  deg1: number
+  min1: number
+  sec1: number
+  deg2: number
+  min2: number
+  sec2: number
+  altitude: number
+  size: number
+  horizontal_precision: number
+  vertical_precision: number
+---
+
+The parameter number types are as follows:
+
+```
 parameter_types:
   name: string
```

Comment:
Instead of specifying the types in a YAML format, can you mention them in prose wherever they’re relevant? (Like the other record types do)

Reply:
Here's an example: https://github.com/StackExchange/dnscontrol/blob/master/documentation/functions/domain/TLSA.md

Reply:
It looks like the yaml format for this builder is still there?